mailbox/store/src/main/java/org/apache/james/mailbox/store/search/MessageSearches.java

```diff
@@ -267,8 +267,7 @@ private boolean isInAttachments(String value, List<MessageAttachmentMetadata> at
             return textExtractor
                     .extractContent(
                         rawData,
-                        // todo we likely want only the media type here
-                        attachment.getType().asString())
+                        attachment.getType())
```

Comment:
so we don't use the charset here?

Reply:
The charset s encoded in the content type.